scripts/api.js

```diff
@@ -11,12 +10,12 @@ class ModuleApi {
         return volume * game.settings.get(MODULE_ID, SETTING_VOLUME);
     }
 
-    static getMacroVariables(messageId, actor) {
-        const message = game.messages.get(messageId);
+    static getMacroVariables(macro) {
         const sourceTokenFallback = canvas.tokens.controlled[0] ?? game.combat?.current?.tokenId;
         const targetsFallback = [...game.user.targets];
+        const flowInfo = macro?.flags?.["lancer-weapon-fx"]?.flowInfo;
```

Comment:
Lancer v2 no longer stores information in chat messages, so we now pass in temporary values through `flags`. This cleans up the macros too.

No functional changes.